core/src/test/java/org/apache/calcite/test/SqlToRelTestBase.java

```diff
@@ -263,6 +263,11 @@ Tester withCatalogReaderFactory(
 
     /** Returns a tester that uses a given context. */
     Tester withContext(UnaryOperator<Context> transform);
+
+    /** Trims a RelNode. */
+    default RelNode trimRelNode(RelNode relNode) {
+      return relNode;
```

Comment:
I'm not quite sure about adding this new `trimRelNode` that by default does nothing (hence does not trim the input RelNode).
Are we sure this is the right place for this method?

Reply:
I didn't find a method in the Tester to only trim the node. I see convertSqlToRel(..) trims the query if we enable trimming. I copied the code from there and put it together as a separate method because i needed to use the trimming part of that function but not conversion from SQL to Rel.
If you think there is a better place to do this, i would be happy to refactor, or if we already have code doing this, i can reuse that.
I will remove the default behavior of this method, since we only have 1 implementation of Tester at the moment.